data/repo_meta/new_java.txt

```diff
@@ -0,0 +1,100 @@
+Snailclimb/JavaGuide
```

Comment:
CodeSearchNet provides the repo list as txt, each line is a repo id `username/repo_name`. I think keeping the same format is easier to work with since we are based on CodeSearchNet.

Reply:
We are using `new_<language>.txt` to denote the list is in addition to CodeSearchNet. I was wondering if we should just append to the CodeSearchNet list. @DataCorrupted

Reply:
I think merging makes more sense. Otherwise, when you are processing these repos you have to open two files instead of one?

Reply:
Right running the script twice on two lists can be troublesome, @ethanbwang we can merge the lists. Since the CodeSearchNet's list is documentated in their paper we can always reference to the paper to know which repo is new. And please prioritize repo searching for python for now, thanks.